### PR TITLE
Simplify Widgets code and Move stuff into Common

### PR DESCRIPTION
**Pull request by bordoni**
🎫 [ECP-486]
---
📹 https://i.bordoni.me/Z4u6L7kY

[ECP-486]: https://theeventscalendar.atlassian.net/browse/ECP-486

### CODE REVIEW

File: src/admin-views/widgets/components/fields/section.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+?>
+<div
+	<?php tribe_classes( $section_classes ); ?>
+	<?php
+	// Not escaped - contains html (data-attr="value").
+	echo $dependency; // phpcs:ignore
+	?>
+>
+	<?php if ( ! empty( $label ) ) : ?>
+		<?php // Note: the actual widget title/handle is an <h3>. ?>
```
Comment (Camwyn) [inline src/admin-views/widgets/components/fields/section.php:36]: No - it's an h2.
So maybe the below one should be an h3?

**Review by Camwyn — CHANGES_REQUESTED** [review]

File: src/admin-views/widgets/components/fields/section.php (hunk 1 above)
Comment (Camwyn) [reply src/admin-views/widgets/components/fields/section.php:36]: This is changing - but better to fix it here first?

File: src/admin-views/widgets/components/fields/section.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<div
+	<?php tribe_classes( $section_classes ); ?>
+	<?php
+	// Not escaped - contains html (data-attr="value").
+	echo $dependency; // phpcs:ignore
+	?>
+>
+	<?php if ( ! empty( $label ) ) : ?>
+		<?php // Note: the actual widget title/handle is an <h2>. ?>
+		<h4 class="tribe-widget-form-control__section-title"><?php echo esc_html( $label ); ?></h4>
```
Comment (Camwyn) [inline src/admin-views/widgets/components/fields/section.php:37]: as part of https://github.com/the-events-calendar/tribe-common/pull/1543#pullrequestreview-587870355
should this be an <h3>?

Comment (bordoni) [reply src/admin-views/widgets/components/fields/section.php:37]: No not in this case, this is the admin.

**Pull request merged**